app/controllers/participants/validations_controller.rb

```diff
@@ -147,13 +145,23 @@ def validate_participant_details_and_redirect
       store_form_and_redirect_to_step :complete
     end
 
+    def participant_details
+      @participant_validation_form.attributes
+        .slice(:trn, :name, :date_of_birth, :national_insurance_number)
+        .transform_keys do |key|
+          case key
+          when :name
+            :full_name
+          when :national_insurance_number
+            :nino
+          else
+            key
+          end
+        end
+    end
```

Comment:
```suggestion
    def participant_details
      @participant_validation_form.attributes.slice(:trn, :name, :date_of_birth, :national_insurance_number)
    end
```

I think I'd do the middle ground, and then closer to what you had in the caller:

```ruby
result = ParticipantValidationService.validate(trn: participant_details[:trn],
                                                     full_name: participant_details[:name],
                                                     date_of_birth: participant_details[:date_of_birth],
                                                     nino: participant_details[:national_insurance_number])
```

What do you think?

Reply:
Fair, your call, I'll leave it up to you.